neo4j/metadata.csv

```diff
@@ -1,37 +1,115 @@
-metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name
-neo4j.arraystore.size,gauge,,unit,,The size of the array store,0,neo4j,Array Store Size

```

Comment:
You should not remove old metrics since there can be users running older versions of the check. What you can do is mention in the description that the metrics is only emitted up to version X of the check

Reply:
bump